application/widget/shell.py

```diff
@@ -44,22 +49,36 @@ def run_shell(self) -> None:
         self.container = self.docker_manager.selected_container
         self.output_widget = self.query_one("#shell-output", RichLog)
         self.input_widget = self.query_one("#shell-input", Input)
+        self.shell = "bash"
```

Comment:
`default_shell = "bash"` or just remove this and set where it is used to "bash".  
I don't have a strong opinion.